packages/core/src/Cardano/types/Governance.ts

```diff
@@ -94,14 +94,14 @@ export type GovernanceAction =
   | NewConstitution
   | InfoAction;
 
-export enum Vote {
+export enum CoreVote {
```

Comment:
We don't export `Certificates` as `CoreCertificates` because we use the syntax `Cardano.Certificate` so I think we should leave this as is, and if you need to make distinctions in other files you could use the `as` syntax when importing.

Reply:
My poor Core types 😭

All types are renamed back @mchappell